frontend/components/admin/distributionConfig.vue

```diff
@@ -1,33 +1,249 @@
+          <v-btn color="red darken-1" flat @click="confirmDelete = false">Cancle</v-btn>
+          <v-btn color="green darken-1" flat @click.prevent.stop="deleteAccount">Submit</v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+
     <v-data-table :headers="headers" :items="items" disable-initial-sort hide-actions>
       <template v-slot:items="props">
-        <td>{{props.item.name}}</td>
-        <td class="text-xs-left">{{ props.item.lockedWallet }}</td>
-        <td class="text-xs-left">{{ props.item.unlockWallet }}</td>
-        <td class="text-xs-left">{{ props.item.ratio | number('0,0.00%')}}</td>
+        <tr>
+          <td>{{props.item.name}}</td>
+          <td class="text-xs-left">{{ props.item.lockedWallet }}</td>
+          <td class="text-xs-left">{{ props.item.unlockWallet }}</td>
+          <td class="text-xs-left">{{ props.item.ratio | number('0,0.00%')}}</td>
+          <td class="justify-center layout px-0">
+            <v-icon small class="mr-2" @click="editDistributionConfigs(props.item)">edit</v-icon>
+            <v-icon small @click="deleteDistributionConfigs(props.item)">delete</v-icon>
+          </td>
+        </tr>
       </template>
     </v-data-table>
   </div>
 </template>
 
 <script>
 import distributionConfigs from './distributionConfigs.mixin'
+import { BigNumber } from 'bignumber.js'
 
 export default {
   mixins: [distributionConfigs],
+  data() {
+    return {
+      mode: '',
+      transectionHash: '',
+      ethLoading: false,
+      valid: true,
+      dialog: false,
+      fieldRules: [v => !!v || 'This field is required'],
+      confirmDelete: false,
+      dialogValue: {
+        name: '',
+        lockedWallet: '',
+        unlockWallet: '',
+        ratio: null,
+        ratioDTV: 0,
+        ratioDecimals: 0
+      }
+    }
+  },
   computed: {
     items() {
       return this.distributionConfigs
     },
     headers() {
       return [
         { text: 'Name', align: 'left', sortable: true, value: 'name' },
-        { text: 'LockedWallet', align: 'left', value: 'lockedWallet', sortable: true },
-        { text: 'UnlockWallet', align: 'left', value: 'unlockWallet', sortable: true },
-        { text: 'Ratio', align: 'left', value: 'ratio', sortable: true }
+        { text: 'LockedWallet', align: 'left', value: 'lockedWallet', sortable: false },
+        { text: 'UnlockWallet', align: 'left', value: 'unlockWallet', sortable: false },
+        { text: 'Ratio', align: 'left', value: 'ratio', sortable: false },
+        { text: 'Action', align: 'center', value: 'action', sortable: false }
       ]
+    },
+    percentage() {
+      console.log(
+        new BigNumber(this.dialogValue.ratioDTV)
+          .dividedBy(Math.pow(10, this.dialogValue.ratioDecimals))
+          .times(100)
+          .toNumber()
+      )
+      return new BigNumber(this.dialogValue.ratioDTV)
+        .dividedBy(Math.pow(10, this.dialogValue.ratioDecimals))
+        .times(100)
+        .toNumber()
```

Comment:
BigNumber is make to replace Number of JS. If you convert it to Number it's useless.
```suggestion
        .toString()
```